R/plugin-awesomeMarkers.R

```diff
@@ -0,0 +1,187 @@
+leafletAwesomeMarkersDependencies <- function() {
+  list(
+    htmltools::htmlDependency(
+      "leaflet-awesomemarkers",
+      "2.0.2",
+      system.file("htmlwidgets/plugins/Leaflet.awesome-markers", package = "leaflet"),
+      script = c('leaflet.awesome-markers.min.js','bootstrap.min.js'),
+      stylesheet = c('leaflet.awesome-markers.css','bootstrap.min.css',
```

Comment:
You can decide how you want to do it. If you want to add it by default, you can; just have this list contain all three htmlDependency objects, instead of just the one.

But you could also detect if "glyphicon" or "fa" is being used, and add just font-awesome or bootstrap instead of both. This wouldn't go in a new place in the code, it would go at the top of `addAwesomeMarkers` where you're already adding stuff to `map$dependencies`.